src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -205,6 +210,22 @@ func (cfg *Config) ApplyDefaults() {
 		cfg.applyClientDefaults()
 	}
 
+	if cfg.ProxiedSites == nil {
+		log.Debugf("Adding empty proxiedsites")
+		cfg.ProxiedSites = &proxiedsites.Config{
+			Delta: &proxiedsites.Delta{
+				Additions: []string{},
+				Deletions: []string{},
+			},
+			Cloud: []string{},
+		}
+	}
+
+	if cfg.ProxiedSites.Cloud == nil || len(cfg.ProxiedSites.Cloud) == 0 {
+		log.Debugf("Loading default cloud proxiedsites")
+		cfg.ProxiedSites.Cloud = defaultProxiedSites
```

Comment:
Here is where we default the proxiedsites list to the default list (generated by genconfig).